homeassistant/components/onewire/binary_sensor.py

```diff
@@ -1,11 +1,17 @@
 """Support for 1-Wire binary sensors."""
+from __future__ import annotations
+
 import os
 
 from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import CONF_TYPE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from .const import CONF_TYPE_OWSERVER, DOMAIN, SENSOR_TYPE_SENSED
-from .onewire_entities import OneWireProxyEntity
+from .onewire_entities import OneWireBaseEntity, OneWireProxyEntity
 from .onewirehub import OneWireHub
 
 DEVICE_BINARY_SENSORS = {
```

Comment:
Define `TypedDict` for `DEVICE_BINARY_SENSORS`, `HOBBYBOARD_EF` and `EDS_SENSORS`.

Reply:
Added in latest commit